bin/tart2ms

```diff
@@ -93,19 +102,25 @@ if __name__ == "__main__":
         logger.info("Raw data is being written to the database per user request")
  
     if ARGS.single_field or ARGS.rephase:
-        if ARGS.rephase == "SCP":
-            phase_center_policy = "rephase-SCP"
-        elif ARGS.rephase == "NCP":
-            phase_center_policy = "rephase-NCP"
-        elif ARGS.rephase == "obs-midpoint":
+        # read special positions from catalogs
```

Comment:
SCP and NCP are special positions now defined along with the rest via included json